openupgrade_scripts/scripts/account/17.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,497 @@
+account      / account.reconcile.model.template / match_amount_max (float)      : DEL
+account      / account.reconcile.model.template / match_amount_min (float)      : DEL
+account      / account.reconcile.model.template / match_journal_ids (many2many) : DEL relation: account.journal
+account      / account.reconcile.model.template / match_label (selection)       : DEL selection_keys: ['contains', 'match_regex', 'not_contains']
+account      / account.reconcile.model.template / match_label_param (char)      : DEL
+account      / account.reconcile.model.template / match_nature (selection)      : DEL required, selection_keys: ['amount_paid', 'amount_received', 'both']
+account      / account.reconcile.model.template / match_note (selection)        : DEL selection_keys: ['contains', 'match_regex', 'not_contains']
+account      / account.reconcile.model.template / match_note_param (char)       : DEL
+account      / account.reconcile.model.template / match_partner (boolean)       : DEL
+account      / account.reconcile.model.template / match_partner_category_ids (many2many): DEL relation: res.partner.category
+account      / account.reconcile.model.template / match_partner_ids (many2many) : DEL relation: res.partner
+account      / account.reconcile.model.template / match_same_currency (boolean) : DEL
+account      / account.reconcile.model.template / match_text_location_label (boolean): DEL
+account      / account.reconcile.model.template / match_text_location_note (boolean): DEL
+account      / account.reconcile.model.template / match_text_location_reference (boolean): DEL
+account      / account.reconcile.model.template / match_transaction_type (selection): DEL selection_keys: ['contains', 'match_regex', 'not_contains']
+account      / account.reconcile.model.template / match_transaction_type_param (char): DEL
+account      / account.reconcile.model.template / matching_order (selection)    : DEL selection_keys: ['new_first', 'old_first']
+account      / account.reconcile.model.template / name (char)                   : DEL required
+account      / account.reconcile.model.template / payment_tolerance_param (float): DEL
+account      / account.reconcile.model.template / payment_tolerance_type (selection): DEL required, selection_keys: ['fixed_amount', 'percentage']
+account      / account.reconcile.model.template / rule_type (selection)         : DEL required, selection_keys: ['invoice_matching', 'writeoff_button', 'writeoff_suggestion']
+account      / account.reconcile.model.template / sequence (integer)            : DEL required
+account      / account.reconcile.model.template / to_check (boolean)            : DEL
+account      / account.report           / _order                        : _order is now 'sequence, id' ('id')
+account      / account.report           / active (boolean)              : NEW hasdefault: default
+# NOTHING TO DO
```

Comment:
Maybe we should to precreate column in pre-migration with default True (and then remove default) like is done in https://github.com/OCA/OpenUpgrade/blob/13.0/addons/account/migrations/13.0.1.1/pre-migration.py#L187.